cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -146,7 +190,13 @@ func (s *Server) Index(name, commit string) error {
 		return s.createEmptyShard(tr, name)
 	}
 
+	opts, err := getIndexOptions(s.Root, nil)
+	if err != nil {
+		return err
+	}
+
 	cmd := exec.Command("zoekt-archive-index",
+		opts.toArgs(),
```

Comment:
I don't think this works. Each argument needs to be its own string. You probably need to make all these args below a slice, then do something like `args = append(args, opts.toArgs()...)`